BUILD.bazel

```diff
@@ -130,17 +130,26 @@ cc_library(
         "include/albatross/utils/MapUtils",
         "include/albatross/utils/RandomUtils",
         "include/albatross/utils/VariantUtils",
+        "tests/mock_model.h",
+        "tests/test_core_distribution.h",
+        "tests/test_covariance_utils.h",
+        "tests/test_models.h",
+        "tests/test_serialize.h",
+        "tests/test_utils.h",
```

Comment:
I think we should extract these files to a new `cc_test_library` and use this library in `albatross-test` and the orion-engine tests.